### PR TITLE
Lorenz

### PR DESCRIPTION
**Pull request by TheLorenzor**
Added home and settings

### CODE REVIEW

**Comment by xnacly** [comment]
Looks good :+1:

**Review by xnacly — APPROVED** [review]

File: src/components/CalenderEntry.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  let newStartTime = new Date(startTime)
+    .toTimeString()
+    .split(' ')[0]
+    .slice(0, 5);
+  let newEndTime = new Date(endTime).toTimeString().split(' ')[0].slice(0, 5);
+
+  return `${newStartTime}-${newEndTime}`;
+}
+
+export default function CalenderEntry({
```
Comment (samuelscheit) [inline src/components/CalenderEntry.tsx:23]: *Calendar

File: src/Service/datastore-service.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import Datastore from 'react-native-local-mongodb';
```
Comment (samuelscheit) [inline src/Service/datastore-service.ts:1]: I recommend using [WatermelonDB](https://github.com/Nozbe/WatermelonDB)

File: ios/StuVReactNativeFrontend3/Info.plist [hunk 3]
```diff
@@ -51,5 +51,24 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>UIAppFonts</key>
+	<array>
+		<string>AntDesign.ttf</string>
```
Comment (samuelscheit) [inline ios/StuVReactNativeFrontend3/Info.plist:56]: preferably only add one icon font

File: src/Service/datastore-service.ts (hunk 2 above)
Comment (Intevel) [reply src/Service/datastore-service.ts:1]: `react-native-local-mongodb` is overblowed. Don't use this.

File: src/components/CalenderEntry.tsx (hunk 1 above)
Comment (xnacly) [reply src/components/CalenderEntry.tsx:23]: resolved in: https://github.com/stuv-mosbach/StuVReactNativeFrontend/commit/fd3538ed298358c905a167a5b704575e6b1118fb

File: src/Service/datastore-service.ts (hunk 2 above)
Comment (xnacly) [reply src/Service/datastore-service.ts:1]: currently we are using asyncstorage

**Pull request merged**